README.md

```diff
@@ -141,10 +141,20 @@ There are several ways to fix this:
     * This behavior can be disabled by setting `ignoreAuthFailures: false` in the config file.
 
 1. How does `xrefcheck` handle redirects?
-    * Permanent redirects (i.e. 301 and 308) are reported as errors.
-    * Temporary redirects (i.e. 302, 303 and 307) are assumed to be valid.
-
-1. How does `xrefcheck` handle localhost links?
+    * The default rules are as follows:
+      * Permanent redirects (i.e. 301 and 308) are reported as errors.
+      * Temporary redirects (i.e. 302, 303 and 307) are assumed to be valid.
+    * The redirect rules can be specified with the `externalRefRedirects` within `networking`, which accepts an array of
+      rules like `{to: "^http://.*", on: temporary, outcome: follow }`. The rule applied is the first one that matches with
+      the `to` and `on` parameters, if any, where
+      * `to` is a regular expression as in `ignoreExternalRefsTo` and its absence means that every link
+        matches.
+      * `on` accepts `temporary`, `permanent` or a specific redirect HTTP code. Its absence also means that
+        every response code matches.
+      * The `outcome` parameter accepts `valid`, `invalid` or `follow`. The last one follows the redirect by applying the
+        same configuration rules.
+
```

Comment:
This comment made me realize that I implemented `to` as if it meant `from`. I am changing it, although it implies that I have also to change some previously implemented tests: I was delaying the check of having a location header in the response after the rules check, and now the location has to be obtained beforehand.

Reply:
Ah, I see.

If you find that it would be easy to remain both `to` and `from` (given that you have both implementations in the git history), please go for it.